### ISSUE TITLE
Magnifiers caret following is not working in the Terminal view

### ISSUE THREAD

**Issue by anthonykozak** [microsoft/vscode#86107]
- VSCode Version: 1.40.2
- OS Version: Windows 10 x64

Steps to Reproduce:

1. Open VS Code and the windows magnifier app (type "magnify" in windows menu)
2. In Magnify settings, check the options to follow keyboard input
3. In VS Code, create a new file, type several lines of text, notice how the magnifier is following properly.
4. Open a terminal view (PowerShell or CMD), and type text, notice how the magnifier cannot follow the input

**Comment by isidorn** [comment]
For the magnifier to work the Terminal would have to have the hidden text input field like the editor does. The textInputField would always be in sync with the content of the Terminal.
@Tyriar assigning to you. We do not currently have the hidden text input field in the terminal correct?

**Comment by Tyriar** [comment]
@isidorn yeah it does uses a hidden text field, it's not in sync due to an old IE bug though. We don't care about IE support anymore so can probably keep it in sync https://github.com/xtermjs/xterm.js/issues/2598

**Comment by isidorn** [comment]
Yes, that would be great!
This would totaly improve the accessibility in the terminal.
Assigning to December and me also so if I find time I might provide a pr. Code pointer welcome

**Comment by Tyriar** [comment]
@isidorn 👍 added some code pointers to https://github.com/xtermjs/xterm.js/issues/2598

**Comment by isidorn** [comment]
Thanks. Let's see if some user picks this up. If not I can jump on it.

**Comment by BuraChuhadar** [comment]
I would like to work on this please if possible.

**Comment by isidorn** [comment]
@BuraChuhadar thanks a lot for jumping on this. I see you already created a PR. Leaving that one to @Tyriar to review.

**Comment by Tyriar** [comment]
PR still needs some iteration, removing from Jan for now

**Comment by isidorn** [comment]
Assigning to June so I look into this if I find time

**Comment by jeanp413** [comment]
I created a PR https://github.com/xtermjs/xterm.js/pull/2996 in xterm to fix https://github.com/xtermjs/xterm.js/issues/2598.

**Comment by isidorn** [comment]
@jeanp413 thanks a lot for looking into this, great job!
